docs/src/pages/components/data-grid/components/CustomPaginationGrid.js

```diff
@@ -35,7 +35,6 @@ export default function CustomPaginationGrid() {
   return (
     <div style={{ height: 400, width: '100%' }}>
       <DataGrid
-        pagination
```

Comment:
> I also took the opportunity to remove pagination prop from DataGrid as I think pagination is enabled by default in DataGrid. Correct me if I am wrong.

I think that it should be reverted. XGrid doesn't have the pagination enabled by default, which would prevent to swap the component and get the same behavior.

Reply:
I thought it would misleading to show non-existent `pagination` prop on `DataGrid`. But happy to revert.

Reply:
Reverted.